O21.Game/RaylibUtils.fs

```diff
@@ -15,36 +20,35 @@ let LoadFontFromMemory (lifetime: Lifetime)
                        (data: byte[])
                        (fontSize: int)
                        (fontChars: int[]): Font =
-    use dataPtr = fixed data
-    use fontCharsPtr = fixed fontChars
+    let dataSpan = Span(data)
+    let fontCharsPtr = Span(fontChars)
+    let font = Font.LoadFromMemory(
+        fileType,
+        dataSpan,
+        fontSize,
+        fontCharsPtr
+    )
```

Comment:
The idea to prevent leaks is to call the constructing function inside of the first delegate passed to `lifetime.Bracket`. Perhaps we can move all three lines to there?